bugwarrior/docs/services/gmail.rst

```diff
@@ -41,6 +41,17 @@ You do not need to specify the ``login_name``, but it can be useful to avoid
 accidentally fetching data from the wrong account. (This also allows multiple
 targets with the same login to share the same authentication token.)
 
+Thread Limit
+-----------------
+
+Gmail search results are capped at 100 max results by default, but may be
+adjusted to meet your needs by specifying `gmail.email_limit`:
```

Comment:
Sorry I missed this before. Need double backticks and we try to keep the docs configuration-language neutral by omitting prefixes.

```suggestion
adjusted to meet your needs by specifying ``email_limit``:
```